tasks/main.yml

```diff
@@ -50,6 +50,7 @@
   tags:
     - auditd
     - selinux
+  when: ansible_distribution_major_version == "7"
```

Comment:
This assumes CentOS, while this role also supports Ubuntu: can you add a "use_audisp" variable to "vars/default.yml" and use that in the when clause?